README.md

```diff
@@ -117,12 +117,27 @@ In addition, the **X-Ldap-Template** header can be used to create complex LDAP s
 
 Suppose, your web resource should only be available for users from `group1` group.
 In such a case you can define `X-Ldap-Template` template as follows:
+
 ```
 proxy_set_header X-Ldap-Template "(&(cn=%(username)s)(memberOf=cn=group1,cn=Users,dc=example,dc=com))";
 ```
 
 The search filters can be combined from less complex filters using boolean operations and can be rather complex.
 
+If your LDAP server doesn't support the memberOf overlay, you can use the following parameters:
+
+```
+proxy_set_header X-Ldap-GroupBaseDN "ou=groups,dc=example,dc=com"
+proxy_set_header X-Ldap-GroupTemplate "(cn=%(groupname)s)"
+proxy_set_header X-Ldap-GroupLimit "group1"
+```
+
+The limit can be done on multiple groups using:
+
+```
+proxy_set_header X-Ldap-GroupLimit "group1,group2"
```

Comment:
Please add trailing semicolon to the nginx directive examples.

Reply:
Hi,
Done :) Thanks!